modules/monitoring/custom/grafana/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# From: https://github.com/grafana/grafana/blob/81957732abf01c9df88307c5f41a12c9f117c0c2/packaging/docker/custom/Dockerfile
+
+ARG GRAFANA_VERSION="latest"
```

Comment:
I would use a generic version ARG like IMAGETAG to use the same array we have for the tags, this way we can sync and build multiple versions with the same modifications

Reply:
I used the original Dockerfile from Grafana, but your suggestion makes a lot of sense.